composer.json

```diff
@@ -209,7 +209,7 @@
         "spryker/products-rest-api": "^2.15.1",
         "spryker/related-products-rest-api": "^1.3.2",
         "spryker/rest-request-validator": "^1.4.0",
-        "spryker/sales-merchant-portal-gui": "^2.0.0",
+        "spryker/sales-merchant-portal-gui": "2.1.0",
```

Comment:
```suggestion
        "spryker/sales-merchant-portal-gui": "^2.1.0",
```

Reply:
In this case installation of the frontend is not successful.